cookbooks/heat_flow/heat-flow-plume.prm

```diff
@@ -0,0 +1,126 @@
+# A model of a plume arriving at the base of a plate.
+
+############### Global parameters
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 1e7
+set Use years in output instead of seconds = true
+set Output directory                       = output-heat-flow-plume
+set Adiabatic surface temperature          = 1600
+
+############### Parameters describing the model
+# Let us here choose again a box domain of size 2x1
```

Comment:
I don't know where this text first came from and what it referred to, but the identical text appears in the composition-reaction and platelike-boundary cookbooks. However, those have a nondimensional geometry, so rather than referring to them, I'm just removing that reference here (and fixing the numbers).

Reply:
👍, thanks for fixing that!